docs/getting-started/development-environment.rst

```diff
@@ -178,7 +178,105 @@ images are having problems building.
 Linux
 ~~~~~
 
-Want to write it? Contact us on the chat!
+Useful packages to install:
+
+.. code-block:: sh
+
+        git:      $ apt install git-all
+
+        zoom:     https://support.zoom.us/hc/en-us/articles/204206269-Installing-or-updating-Zoom-on-Linux#h_adcc0b66-b2f4-468b-bc7a-12c182f354b7
+
+        discord:  https://doc.ubuntu-fr.org/discord
+        
+        VS code:  https://code.visualstudio.com/download
+
+If during the installation you encounter broken packages, try the follwoing command:
+
+.. code-block:: sh
+        $ apt --fix-broken install
+
+
+**Python**
+
+Invenio is developed using Python and JavaScript. 
+If you want to check which version of Python you have, try the following:
+
+.. code-block:: sh
+        # Check the system Python version
+        $ python --version
+
+        # Check the Python 2 version
+        $ python2 --version
+
+        # Check the Python 3 version
+        $ python3 --version
+
+To install Python 3.8 type the following commands:
+
+.. code-block:: sh
+
+        $ sudo apt-get update
+        $ sudo apt-get install python3.8 python3-pip
+
+In the follwoing list you can check if your system has the necessary
+requirements. We highly recommend that you install ``pyenv`` and ``nvm`` - both tools manage 
+version of python and node respectively.
+https://inveniordm.docs.cern.ch/install/requirements/
+
+After installing docker you might have some errors related to permissions, try the following command: 
```

Comment:
It is maybe better to link to useful resources rather than provide these solutions because they tend to became obsolete very quickly and I would pay extra care suggesting to change permissions ;)
What about if you change to something similar to:

```
If you get the following error after installing Docker and running simple commands:
Got permission denied ... /var/run/docker.sock: connect: permission denied
see [here](https://stackoverflow.com/questions/48568172/docker-sock-permission-denied) for some tips on how to solve it.
```

Of course this is Markdown, please change it to Sphinx ;)